### ISSUE TITLE
update broke imports

### ISSUE THREAD

**Issue by trusktr** [solidjs/solid-playground#108]
An update within the past two days broke imports. For example the following sample,

https://playground.solidjs.com/?hash=589409240&version=1.3.16

now shows the error

```
Failed to fetch dynamically imported module: data:text/javascript;charset=utf-8,import%20%7B%20createComponent%20%7D%20from%20'https%3A%2F%2Fcdn.skypack.dev%2Fsolid-js%401.3.16%2Fweb'%3B%0Aimport%20%7B%20render%20%7D%20from%20'https%3A%2F%2Fcdn.skypack.dev%2Fsolidjs%2Fweb'%3B%0Aimport%20html%20from%20'https%3A%2F%2Fcdn.skypack.dev%2Fsolidjs%2Fhtml'%3B%0A%0Afunction%20Counter()%20%7B%0A%20%20return%20html%60%0A%09%09%09%09%3Clumescene%0A%09%09%09%09%09id%3D%22scene%22%0A%09%09%09%09%09perspective%3D%22800%22%0A%09%09%09%09%09webgl%0A%09%09%09%09%09shadowmaptype%3D%22pcfsoft%22%0A%09%09%09%09%09physicallycorrectlights%0A%09%09%09%09%3E%0A%09%09%09%09%09%3Clumenode%20size%3D%22160%20200%2015%22%20mountpoint%3D%220.5%200.5%22%20alignpoint%3D%220.5%200.5%22%3E%0A%09%09%09%09%09%09%3Clumecamerarig%0A%09%09%09%09%09%09%09active%0A%09%09%09%09%09%09%09initialpolarangle%3D%2230%22%0A%09%09%09%09%09%09%09minpolarangle%3D%2250%22%0A%09%09%09%09%09%09%09maxpolarangle%3D%2250%22%0A%09%09%09%09%09%09%09minhorizontalangle%3D%2250%22%0A%09%09%09%09%09%09%09maxhorizontalangle%3D%2250%22%0A%09%09%09%09%09%09%09initialdistance%3D%22500%22%0A%09%09%09%09%09%09%09maxdistance%3D%229000%22%0A%09%09%09%09%09%09%09mindistance%3D%22200%22%0A%09%09%09%09%09%09%09alignpoint%3D%220.5%200.5%22%0A%09%09%09%09%09%09%3E%0A%09%09%09%09%09%09%3C%2Flumecamerarig%3E%0A%0A%09%09%09%09%09%09%3Clumebox%0A%09%09%09%09%09%09%09id%3D%22box%22%0A%09%09%09%09%09%09%09shininess%3D%22100%22%0A%09%09%09%09%09%09%09color%3D%22white%22%0A%09%09%09%09%09%09%09sizemode%3D%22proportional%20proportional%22%0A%09%09%09%09%09%09%09size%3D%221%201%201%22%0A%09%09%09%09%09%09%09note%3D%22align%20point%20needs%20to%20be%20determined.%20Will%20it%20be%20a%20percentage%20or%20absolute%20value%20from%20the%20back%20of%20the%20frame%3F%20Or%3F%22%0A%09%09%09%09%09%09%09alignpoint%3D%220%200%200.4%22%0A%09%09%09%09%09%09%3E%3C%2Flumebox%3E%0A%09%09%09%09%09%3C%2Flumenode%3E%0A%09%09%09%09%3C%2Flumescene%3E%0A%09%09%09%60%3B%0A%7D%0A%0Awindow.dispose%20%3D%20render(()%20%3D%3E%20createComponent(Counter%2C%20%7B%7D)%2C%20document.getElementById(%22app%22))%3B%0A
```

**Comment by edemaine** [comment]
I don't see an NPM package named `solidjs`.  `import { render } from "solid-js/web";` seems to work.  I'm not sure what `solidjs/html` is supposed to be (haven't used HyperScript before).

**Comment by trusktr** [comment]
@modderme123 looks like something deleted the `-`.

**Comment by trusktr** [comment]
I tested making a new link with the dashes added back:

https://playground.solidjs.com/?hash=-948883626&version=1.3.16

Abd it works.

I think I know what happened actually.

Somehow, now that I recall, I replaced all `"-"` in the editor with `""` and the example somehow continued to work after the imports became incorrect, so u was under the impression that it worked, and I made a hash and saved the demo.

Then when I returned fresh later, it didn't work (as we now expect).
